res/layout/conversation_item_received.xml

```diff
@@ -151,6 +151,23 @@
                 app:scaleEmojis="true"
                 tools:text="Mango pickle lorem ipsum"/>
 
+            <Button
+                android:id="@+id/show_full_message"
+                android:visibility="gone"
+                android:layout_width="wrap_content"
+                android:layout_height="28dp"
+                android:layout_gravity="center_vertical"
+                android:layout_marginLeft="10dp"
+                android:layout_marginRight="10dp"
+                android:layout_marginTop="6dp"
+                android:layout_marginBottom="10dp"
+                android:paddingLeft="12dp"
+                android:paddingRight="12dp"
+                android:background="@drawable/conversation_item_update_background"
```

Comment:
The button somehow looks too... grey to me; the following looks better IMO but not great still:

```suggestion
                android:background="@color/delta_primary"
                android:textColor="@color/white"
                tools:visibility="visible"
```

Reply:
yip, the button could need some tweaking, however, also outgoing buttons need tweaking, and by just saying `@color/delta_primary`, the rounded corners are lost.

Reply:
For me it looks equally good with round and square corners, but yes, if we want to preserve rounded corners, this needs some more work.

What do you mean with "outgoing buttons"?

Reply:
> What do you mean with "outgoing buttons"? 

the ones defined in `res/layout/conversation_item_sent.xml`. we have two layout for the bubbles, one incoming, one outgoing. you suggestion would only change the incoming ones (on the left side).

however, to move forward, i'd like to put that layout discussion into another pr, current state is not bad.